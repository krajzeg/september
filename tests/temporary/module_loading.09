print("If runtime.sept builds and loads correctly, this will be True:", runtimeLoaded)
