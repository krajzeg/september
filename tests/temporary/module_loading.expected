If runtime.sept builds and loads correctly, this will be True: <True>
